### ISSUE TITLE
Time zones

### ISSUE THREAD

**Issue by jeffblake** [braintree/braintree_ruby#161]
cc: https://github.com/braintree/braintree_ruby/issues/42

There's also no way to get the time zone of the merchant via the API, so I'm currently hard-coding CST... and the time zone is not changeable via the control panel.

Docs here say `first_billing_date` is a `Time` object, but the SDK is giving me a string. 
https://developers.braintreepayments.com/reference/request/subscription/create/ruby#first_billing_date

**Comment by jaypatel512** [comment]
Hey @jeffblake !

About `first_billing_date` being a Time object, currently we only return string back, and cannot change it right now without making a breaking change. We will definitely fix this in our next major version bump. We will fix the docs right away changing that to string.

**Comment by mandysimon88** [comment]
@jaypatel512 This applies to all the dates coming back, include for example @paid_through_date on Subscription, @next_billing_date, @billing_period_end_date, @billing_period_start_date...

It wouldn't be so bad if it returned a DateTime string, because then one could parse it back into the object. But just returning a Date string opens the door to... well, you know.

But, we can fill in the details around, with your help, as a temporary patch.

If I create a Subscription at `2018-02-15 16:03:39 UTC`, recurring monthly, what would the date/time stamp of `paid_through_date` look like, if indeed it were sending back the time as well as the date? (I'm assuming recurring billing actually operates off of DateTime but is only telling me the Date.)

If you can give me the DateTime would be for that one example, I can extrapolate to figure out how to deal with no Time while we wait on a fix.

Thank you!

**Comment by darwinup** [comment]
The recurring monthly date and time stamp would be 2018-03-17   16:03:39

**Comment by hollabaq86** [comment]
👋  Version 3.0.1 includes updates so that date-like responses from our API that do not include timestamps are set as Date class objects. 

The original issue here is around certain fields not including timezone information, and unfortunately we can't update the API to include timezones.

We're going to close this issue. If anyone has future questions or feedback, please open a new issue.
